backend/danswer/connectors/web/connector.py

```diff
@@ -104,7 +107,7 @@ def load_from_state(self) -> GenerateDocumentsOutput:
                             sections=[Section(link=current_url, text=page_text)],
                             source=DocumentSource.WEB,
                             semantic_identifier=current_url.split(".")[-1],
-                            metadata={},
+                            metadata={"time visited": current_visit_time},
```

Comment:
nit: should we have a consistent format for the metadata? Here it's lowercase, for confluence it was uppercase (in my productboard pr it's snake case)